Client/mods/deathmatch/logic/luadefs/CLuaClientDefs.cpp

```diff
@@ -80,3 +82,20 @@ bool CLuaClientDefs::IsCapsLockEnabled()
 {
     return ((::GetKeyState(VK_CAPITAL) & 0x0001) != 0);
 }
+
+bool CLuaClientDefs::SetCursorColor(float r, float g, float b, float alpha) noexcept
+{
+    if (!g_pCore->IsMenuVisible())
+        g_pCore->GetGUI()->SetCursorColor(r, g, b, alpha);
```

Comment:
Could you make the expected alpha argument in range 0-255, and not 0-1, like [setCursorAlpha](https://wiki.multitheftauto.com/wiki/SetCursorAlpha)

Reply:
having alpha makes more sense to set color and alpha while using setCursorAlpha is up to u
I will make it optional argument

Reply:
Sure make optional arg

Reply:
But for the developer, alpha should be 0-255